### ISSUE TITLE
Support for MacOS with processor M

### ISSUE THREAD

**Issue by wilebaldo** [Qendolin/better-clouds#24]
Could you add support for MacOS with processor AppleSilicon, M1 M2?

**Comment by Qendolin** [comment]
I've actually already started to work on this for the 1.20 release but it's possible to support 100% of the features. Apple really sucks for only supporting such an old OpenGL Version :(

**Comment by wilebaldo** [comment]
Ok, I will wait. Thanks a lot.

**Comment by wilebaldo** [comment]
https://mclo.gs/dD2sw50

[latest.log](https://github.com/Qendolin/better-clouds/files/11818218/latest.log)

I'm not sure witch one is the correct log file, but I give your this 2

**Comment by Qendolin** [comment]
Thanks, looks like the I still have some issues to fix :/

**Comment by Qendolin** [comment]
Can you try again with the version contained in this zip file: 
[better-clouds-1.2.3-BETA+1.20.1.e1e46a3.zip](https://github.com/Qendolin/better-clouds/files/11825065/better-clouds-1.2.3-BETA%2B1.20.1.e1e46a3.zip)

**Comment by wilebaldo** [comment]
Sorry the delay on answer I was working. here is the Log.

[latest.log](https://github.com/Qendolin/better-clouds/files/11827428/latest.log)

**Comment by Qendolin** [comment]
No problem I also quite busy, but it seems like you forgot to add the mod? It is not being loaded. Did you maybe forget to unzip it first?

**Comment by wilebaldo** [comment]
Yeah! you are right, I forgot to unzip the mod, here is the new log file.

[latest.log](https://github.com/Qendolin/better-clouds/files/11827829/latest.log)

**Comment by wilebaldo** [comment]
Just give you a little be more information, the first beta the game crash when I go to the world, in the secound this is how you see the graphics.
![2023-06-21_18 36 47](https://github.com/Qendolin/better-clouds/assets/8034131/1ea420c5-aac5-40ba-8424-846628d07b44)

**Comment by Qendolin** [comment]
I gotta say, that is very odd. I think I might try a completely different approach for supporting older OpenGL versions.

**Comment by Qendolin** [comment]
Maybe it is fixed now: https://modrinth.com/mod/better-clouds/version/1.3.2+1.20.1-B

**Comment by lavndor** [comment]
> Maybe it is fixed now: https://modrinth.com/mod/better-clouds/version/1.3.2+1.20.1-B

Seems to be working on M1 Pro/macOS Sonoma

**Comment by wilebaldo** [comment]
> Maybe it is fixed now: https://modrinth.com/mod/better-clouds/version/1.3.2+1.20.1-B
Is working, perfect. This is awesome.

![2023-06-28_22 29 23](https://github.com/Qendolin/better-clouds/assets/8034131/2ef2d068-a990-4071-a9d7-c5f625cc4a4c)

**Comment by Qendolin** [comment]
Thank you so much for helping me fix this issue and thank you for the donation, I really appreciate it!!
